src/pages/profile/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { NextPage } from "next";
+import Link from "next/link";
+import { Instagram, Facebook } from "react-feather";
+
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const ContactUs: NextPage = () => {
+  return (
+    <Page
+      bgImage="bernard"
+      title="Yeboi it's me Bernard"
+      description=""
+      subBanner
+      renderSubcontent={() => {
```

Comment:
What does `renderSubcontent` do and how does it differ from <Page> subcontent here </Page>

Reply:
Renders the grey thing below the banner picture (technically still part of banner but oh welpz). Hem is the difference only better abstraction? Since you can use the same banner construct for the rest of the pages.

Reply:
iya wkwk